src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -62,9 +62,9 @@ void Unit::UpdateDamagePhysical(WeaponAttackType attType)
     float totalMin = 0.f;
     float totalMax = 0.f;
 
+    float tmpMin, tmpMax;
```

Comment:
I get that you copied it from the line below, but why are we not initializing the variables?

Reply:
They'll get override by CalculateMinMaxDamage

Reply:
Got it.